src/mlpack/methods/pca/pca.hpp

```diff
@@ -27,10 +27,11 @@ namespace pca {
  * common, widely-used technique that is often used for either dimensionality
  * reduction or transforming data into a better basis.  Further information on
  * PCA can be found in almost any statistics or machine learning textbook, and
- * all over the internet.
+ * all over the internet. Note this class will be changed to have the name PCA
+ * in mlpack 3.0.0
```

Comment:
This reverts some changes that we have made for mlpack 3.0.0, can you undo your changes to this file please?